integration/client-data-test.ts

```diff
@@ -955,7 +956,9 @@ test.describe("Client Data", () => {
       expect(html).not.toMatch("Should not see me");
 
       expect(logs).toEqual([
-        expect.stringContaining("Download the React DevTools"),
+        ...(browserName === "webkit"
+          ? []
+          : [expect.stringContaining("Download the React DevTools")]),
```

Comment:
the react devtools message doesn't show on safari as it's not a supported browser :man_shrugging:
https://github.com/facebook/react/blob/7b7fac073d1473df839a1caf8d0444c32bf4de49/packages/react-dom/src/client/ReactDOMClient.js#L58-L63